README.md

```diff
@@ -1,31 +1,121 @@
 
-A [Visual Studio Code](https://code.visualstudio.com/)
-[extension](https://marketplace.visualstudio.com/VSCode) that aims to allow
-users of all technical backgrounds to effectively use [DVC](https://dvc.org/).
-
-## Current State
-
-We want early adopter feedback! Please let us know what you like and don't like
-about the extension. Feel free to reach out in this repository's
-[issues](https://github.com/iterative/vscode-dvc/issues) or via any of the other
-existing [support channels](https://dvc.org/support).
-
-## Quick Start
-
-- **Step 1.**
-  [Install a supported version of DVC on your system](https://dvc.org/doc/install)
-- **Step 2.**
-  [Install the DVC extension for Visual Studio Code](https://code.visualstudio.com/docs/editor/extension-gallery).
-- **Step 3.** See Walkthrough.
+Run, compare, visualize, and track machine learning experiments right in VS
+Code. This extension uses [DVC](https://dvc.org/), an open-source data
+versioning and ML experiment management tool. No additional services or
+databases are required.
+
+![Extension Overview](https://raw.githubusercontent.com/iterative/vscode-dvc/main/extension/docs/overview.gif)
+
+- **Experiment tracking**: Record training data, parameters, and metrics on top
+  of [Git](https://git-scm.com/). Navigate your experiments, compare their
+  results, and find the best ML models.
+- **Visualization**: Plot performance data in a customizable dashboard including
+  one or more overlaid experiments.
+- **Live tracking**: Capture and see metrics changing in real time.
+- **Reproducibility**: Make sure that anyone can recover or confirm previous
+  experiments, and run new experiments based on their results.
+- **Data Management**: Handle and version large datasets, files, and models
+  effectively right from VS Code.
+
+> **Note**: We are in public beta and welcome early adopter feedback!  
+> Feel free to reach out [via Discord] or open [issues in GitHub].
+>
+> [issues in github]: https://github.com/iterative/vscode-dvc/issues
+> [via discord]: https://discord.gg/BGCjJHvDHt
+
+### Why prefer this extension?
+
+- Enjoy the best developer experience with the first experiment tracking
+  interface for an IDE.
+- No external servers, databases, subscriptions, etc. Data stays fully under
+  your control and your existing Git hosting is used to share and collaborate.
+- Go beyond a simple metrics dashboard with complete ML experiments that include
+  metrics, code, and data. Powered by [DVC experiment
+  versioning][dvc experiments].
+- Implement [data versioning] on top of your favorite cloud storage, such as
+  Amazon S3, Azure Blob Storage, Google Cloud Storage, NFS, etc.
+
+[dvc experiments]:
+  https://dvc.org/doc/user-guide/experiment-management/experiments-overview
+[data versioning]: https://dvc.org/doc/use-cases/versioning-data-and-model-files
+
+## Quick start
```

Comment:
Add a ToC? Unlike in GH, https://marketplace.visualstudio.com/items?itemName=Iterative.dvc doesn't auto-generate one nor does it add 🔗 anchors per header.